src/components/page-header.less

```diff
@@ -0,0 +1,7 @@
+.page-header-container{
+    .nav-bar{
+        background-color: #010101!important;
```

Comment:
for the future, let's find out if we can save the color as a variable and reuse it